src/go/pt-galera-log-explainer/display/timelinecli.go

```diff
@@ -0,0 +1,352 @@
+package display
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"sort"
+	"strings"
+
+	// regular tabwriter do not work with color, this is a forked versions that ignores color special characters
+	"github.com/Ladicle/tabwriter"
+	"github.com/percona/percona-toolkit/src/go/pt-galera-log-explainer/types"
+	"github.com/percona/percona-toolkit/src/go/pt-galera-log-explainer/utils"
+)
+
+// TimelineCLI print a timeline to the terminal using tabulated format
+// It will print header and footers, and dequeue the timeline chronologically
+func TimelineCLI(timeline types.Timeline, verbosity types.Verbosity) {
+
+	timeline = removeEmptyColumns(timeline, verbosity)
+
+	// to hold the current context for each node
+	// "keys" is needed, because iterating over a map must give a different order each time
+	// a slice keeps its order
+	keys, currentContext := initKeysContext(timeline)    // currentcontext to follow when important thing changed
+	latestContext := timeline.GetLatestContextsByNodes() // so that we have fully updated context when we print
+	lastContext := map[string]types.LogCtx{}             // just to follow when important thing changed
```

Comment:
Minor: In PMM we are following this advise: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style


So I suggest to create empty maps like this:
```suggestion
	lastContext := make(map[string]types.LogCtx)              // just to follow when important thing changed
```

Reply:
Agree, because actually I could prealloc the correct size here